plans/no-rpm.fmf

```diff
@@ -0,0 +1,28 @@
+execute:
+    how: tmt
+prepare:
+    - name: packages
+      how: install
+      package: [ golang, podman-docker ]
+    - name: enable-linger
+      how: shell
+      script: loginctl enable-linger $(whoami)
+    - name: golangci
+      how: shell
+      script: |
+        # https://golangci-lint.run/welcome/install/
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/HEAD/install.sh | sh -s -- -b /bin v2.1.6
```

Comment:
Should this read the version from `Makefile`? On this branch it’s a different major version, with a different config file format.

Reply:
I'll check on this, thanks